tests/deviation.rs

```diff
@@ -12,12 +12,12 @@ use std::f64;
 fn test_count_eq() -> Result<(), MultiInputError> {
     let a = array![0., 0.];
     let b = array![1., 0.];
-    let c = array![0., 1.];
+    let mut c = array![0., 1.];
```

Comment:
Suggestion: create a dedicated test case for ownership variation.

The individual tests for each function, towards top of this file, focus on the desired behaviour mathematically. Towards the end of the file the tests focus on shared issues which make them a consistently behaving module. 

So my suggestion is to create a test focused on the issue of ownership variation because it's a quality that all the function in this module need to share.

Reply:
You always get me when I am being lazy. I'll refactor :crossed_fingers:

Reply:
Done - check it out @munckymagik :+1: